packages/autocomplete-preset-algolia/src/highlight/reverseHighlightedParts.ts

```diff
@@ -9,6 +9,6 @@ export function reverseHighlightedParts(parts: ParsedAttribute[]) {
 
   return parts.map((part, i) => ({
     ...part,
-    isHighlighted: !isPartHighlighted(parts, i),
+    isHighlighted: !isPartHighlighted(parts, i) && part.value.trim().length > 0,
```

Comment:
if you forcibly not want to highlight spaces, why isn't this logic in isPartHighlighted? I feel like something as "`this is` a match" seems nicer than "`this` `is` a match" (less changes in background, thus less visual noise)

Reply:
I'll cover these cases with tests separately from this PR 👍 

c150a0170b02e99269712f3564486a9644dbe768